packages/block-library/package.json

```diff
@@ -79,7 +79,9 @@
 	"sideEffects": [
 		"build-style/**",
 		"src/**/*.scss",
-		"{src,build,build-module}/*/init.js"
```

Comment:
The `init.js` files for individual blocks are still problematic. As far as I know, they are not used anywhere in the Gutenberg app. And the built package bundle (`build-module/index.js`) doesn't include them, because they are not imported anywhere.

They are exposed as package exports by virtue of the `"./build-module/*"` export in `package.json`. These exports are currently broken, however, because we don't build them as separate entrypoints yet.